api/opencl_architecture.asciidoc

```diff
@@ -789,13 +792,19 @@ memory:
 
 The constant address space is disjoint from the generic address space.
 
+NOTE: The generic address space is <<unified-spec, missing before>> version
```

Comment:
Strictly speaking, generic address space **is** documented by [OpenCL 1.2 spec](https://www.khronos.org/registry/OpenCL/specs/opencl-1.2.pdf), from section 6.5 Address Space Qualifiers:

> If the type of an object is qualified by an address space name, the object is allocated in the
specified address name; otherwise, the object is allocated in the generic address space.

> The generic address space name for arguments to a function in a program, or local variables of a
function is `__private`. 

But apparently it has different meaning, so, not sure if we need to change anything here

Reply:
I hadn't ever made that connection before.  I think this is just a coincidence of wording and that the use of `generic address space` in that paragraph could have also just said `private address space` directly.

In the context of a unified specification I think the current text is clear enough, it is talking about an actual address space called "generic", and that is what was missing before 2.0.

I will resolve this comment, but if you think the wording should be changed then please file an issue and I'll happily look at it a bit more.